pnc/src/main/java/org/jboss/pnc/bacon/pnc/client/BifrostClient.java

```diff
@@ -40,7 +40,7 @@ public void writeLog(String id, boolean follow, Consumer<String> onLine) throws
         }
 
         URI logsUrl = baseUrl.resolve(URI.create("/text?" + query));
-        logger.debug("Reading logs from {}.", logsUrl.toString());
+        logger.debug("Reading logs from {}", logsUrl);
```

Comment:
`logger` has to be changed to `log` here, doesn't it?

Reply:
@michalszynkiewicz Fixed